diff --git a/kedut-webapp/src/main/java/com/kedut/directory/service/AddressService.java b/kedut-webapp/src/main/java/com/kedut/directory/service/AddressService.java
index abe5fcb..2ea9a5e 100644
--- a/kedut-webapp/src/main/java/com/kedut/directory/service/AddressService.java
+++ b/kedut-webapp/src/main/java/com/kedut/directory/service/AddressService.java
@@ -18,10 +18,6 @@
 	
 	
 	
-	public AddressService() {
-		super();
-		// TODO Auto-generated constructor stub
-	}
 
 	@Autowired
 	private AddressDao addressDao;